README.md

```diff
@@ -1,12 +1,12 @@
-The Exchange Union Daemon ([`xud`](https://github.com/ExchangeUnion/xud)) is the reference node implementation powering [OpenDEX](https://opendex.network), a decentralized exchange (DEX) built on top of the [Lightning](https://lightning.network/) and [Raiden](https://raiden.network/) network. `xud` brings individual traders, market makers and exchanges onto OpenDEX to form a single global trading network and liquidity pool.
+The Exchange Union Daemon ([`xud`](https://github.com/ExchangeUnion/xud)) is the reference implementation powering [OpenDEX](https://opendex.network), a decentralized exchange built on top of the [Lightning](https://lightning.network/) and [Raiden](https://raiden.network/) network. `xud` brings individual traders, market makers and exchanges onto OpenDEX to form a single global liquidity pool.
 
-**Traders** use [OpenDEX-enabled exchanges](https://opendex.network/trade/exchanges) as gateways to trade on OpenDEX. These exchanges are usually non-custodial and abstract away from the technological complexity of OpenDEX and offer a superior UX by enabling trading via a simple website. **Exchanges** use `xud` to hedge user trades on OpenDEX in order to lock in profits. **Market Makers** use `xud` to arbitrage with external exchanges (e.g. Binance) and provide liquidity on OpenDEX for a premium. Exchange Union provides SDK's for both, exchanges to automate hedging with `xud` and for market makers to connect to external exchanges and automate the arbitrage trades.
+**Traders** use [OpenDEX-enabled exchanges](https://opendex.network/trade/exchanges) as gateways to trade on OpenDEX. These exchanges are usually non-custodial and abstract away from the technological complexity of OpenDEX and offer a superior UX by enabling trading via a simple website. **Exchanges** use `xud` to hedge user trades using the liquidity on OpenDEX. **Market Makers** use `xud` to offer liqudidity on OpenDEX with a premium leveraging arbitrage to external exchanges like Binance. Exchange Union provides SDK's for both, exchanges to automate hedging with `xud` and for market makers to connect to external exchanges and automate the arbitrage trades.
```

Comment:
~~abstract away~~ abstracted away?

~~liqudidity~~ liquidity on OpenDEX with a premium, leveraging 

These exchanges are usually non-custodial and abstract away from the technological complexity of OpenDEX and offer a superior UX by enabling trading via a simple website

might be better worded as;

These exchanges are hosted on a server by an exchange operator. Under the hood, it interacts with the OpenDex protocol and abstracts all the complexity that comes with it into a nice user interface for the end-users. 

Exchange Union provides SDK's for both, exchanges to automate hedging with `xud` and for market makers to connect to external exchanges and automate the arbitrage trades.

might be better worded as;
 
Exchange union provides necessary toolings that help decrease the friction for both sides of the marketplace. Exchanges can benefit from the SDK that can be used to automate the hedging with `xud` and market makers can use another API  tool that can help them arbitrage by connecting to the external exchanges.